.github/CODEOWNERS

```diff
@@ -85,3 +85,4 @@ sycl/doc/ @pvchupin @kbobrovs
 sycl/doc/extensions/ @mkinsner @jbrodman
 
 xpti/ @tovinkere @andykaylor
+xptifw/ @tovinkere
```

Comment:
We need a back-up person. @andykaylor, are you okay to review patches to `xptifw`?

Reply:
> We need a back-up person. @andykaylor, are you okay to review patches to `xptifw`?

Yes.